sos/report/plugins/ssh.py

```diff
@@ -80,8 +80,21 @@ def user_ssh_files_permissions(self):
         """
         users_data = pwd.getpwall()
 
+        fs_mount_info = {}
+        try:
+            with open('/proc/mounts', "r", encoding='UTF-8') as mounts_file:
+                for line in mounts_file:
+                    (fs_file, fs_vstype) = line.split()[1:3]
+                    fs_mount_info[fs_file] = fs_vstype
+        except Exception:
+            self._log_error("Couldn't read /proc/mounts")
+            return
+        non_local_fs = {'nfs', 'nfs4', 'autofs'}
         # Read the home paths of users in the system and check the ~/.ssh dirs
         for user in users_data:
+            if user.pw_dir in fs_mount_info and \
+                    fs_mount_info[user.pw_dir] in non_local_fs:
+                continue
```

Comment:
We should log something so that end users aren't left wondering why this wasn't collected when previously they may have been expecting it.

Reply:
What verbosity level? Info? (or debug? isnt it too hidden..?)

Reply:
With warn I get this:

```
 Setting up archive ...
 Setting up plugins ...
[plugin:ssh] Skipping capture in /home/jcastill because it's a remote directory
 Running plugins. Please wait ...

  Starting 1/1   ssh             [Running: ssh]
```

Reply:
Yeah, and I dont know if that is not too "vocal". I think info verbosity is the best, but I am ok with any.

Reply:
You are right, in the specific scenario of tens or hundreds of NFS mounted homedirs, the output would be huge.

Reply:
Exactly. I'm pushing a change with info now